locales/es_ES.json

```diff
@@ -0,0 +1,107 @@
+          "dark": " Modo oscuro",
+          "light": " Modo claro",
+          "subheading": "La lente a una nueva era de gráficos en minecraft",
+          "or": "o",
+          "continue": "Seguir leyendo",
+          "heading1": "¿Por qué Iris?",
+          "subheading1": "Esto es lo que hace a Iris diferente",
+          "card1": "Gran compatibilidad con mods",
+          "carddesc1": "Iris está comprometida a asegurar la compatibilidad con mods como prioridad, ¡y ya es compatible con muchos de tus mods favoritos!",
+          "card2": "Diseño Moderno",
+          "carddesc2": "Iris ha sido construida desde el principio para ser extendible y fácil de actualizar. ¡Di adiós a semanas de espera para usar shaders en la nueva actualización de Minecraft!",
+          "card3": "Libre y de Código Abierto",
+          "carddesc3": "Iris es un mod de código abierto al que cualquiera puede contribuir. Esto permite a Iris mejorar al facilitar la colaboración.",
+          "heading2": "Características",
+          "subheading2": "Lo que Iris tiene que ofrecer",
+          "image1": "Paridad con OptiFine",
+          "imagedesc1": "Iris apunta a tener soporte completo para todos los shaderpacks de OptiFine pasados y actuales, ¡y el número de packs soportados por Iris aumenta constantemente!<span class='text-span-2'><br>Captura por Niko</span>",
+          "image2": "Works with Sodium",
+          "imagedesc2": "Iris es compatible con los mods de optimización de CaffeineMC, como Sodium y Phosphor, permitiendo gráficos impresionantes con FPS comparables o incluso mejores que en vanilla.<br><span class='text-span-2'>Captura por Guardian</span>",
+          "heading3": "La ventaja de Iris",
+          "subheading3": "Nuevas técnicas de renderizado que te ponen en la delantera del juego",
+          "shadows": "Sombras",
+          "entities": "Entidades",
+          "screenshottitle1": "Sombras Optimizadas para el Rendimiento",
+          "screenshotdesc1": "Iris usa una técnica llamada Frustum Culling de Sombras para reducir los costes de uno de los mayores consumidores de rendimiento de un shader - las sombras. Usando el frustrum culling de Iris, serás capaz de obtener más altos FPS sin sacrificar la calidad de la iluminación volumétrica o las sombras.<br><span class='text-span-2'>Captura por FoundationGames</span>",
+          "screenshottitle2": "Renderizado de Entidades Eficiente",
+          "screenshotdesc2": "Iris mejora la forma en que Minecraft renderiza entidades y bloques con entidades como cofres, cerdos, o marcos para objetos. Este truco funciona hasta con los shaders desactivados, y en mundos con muchas entidades, puede hasta mejorar el rendimiento en vanilla por hasta un 60 por ciento. La renderización de sombras encuentra una mejora especialmente grande, ¡haciendo zonas que una vez fueron injugables con shaders sitios donde tus FPS són sorprendientemente estables!<br><span class='text-span-2'>Captura por kinzuu_music</span>",
```

Comment:
Field "stillcurious" should be after this line to be consistent with e703134fa756bdbd8832c6ce47d8611aa4fd1e24 from #23